### PR TITLE
Odpdown

### PR DESCRIPTION
**Pull request by vandenoever**

### CODE REVIEW

File: pkgs/tools/typesetting/odpdown/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, buildPythonPackage, libreoffice, lpod, lxml, mistune, pillow
+, pygments
+}:
+
+buildPythonPackage rec {
+
+  name = "odpdown";
```
Comment (fpletz) [inline pkgs/tools/typesetting/odpdown/default.nix]: The name attribute must have the version. Please use `"odpdown-0.4.1"`.

File: pkgs/tools/typesetting/odpdown/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  propagatedBuildInputs = [ libreoffice lpod lxml mistune pillow pygments ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/thorstenb/odpdown;
+    repositories.git = https://github.com/thorstenb/odpdown.git;
+    description = "Create nice-looking slides from your favourite hacker's editor";
+    longDescription = ''
+    Have a tool like pandoc, latex beamer etc, that you can write (or auto-generate) input for within your favourite hacker's editor, and generate nice-looking slides from. Using your corporation's mandatory, CI-compliant and lovely-artsy Impress template. Including syntax-highlighted code snippets of your latest hack, auto-fitted into the slides.
```
Comment (fpletz) [inline pkgs/tools/typesetting/odpdown/default.nix]: Please indent and use newlines in this string.

**Comment by vandenoever** [comment]
@fpletz done

**Comment by fpletz** [comment]
Thanks!

**Pull request merged**